editions/tw5.com/tiddlers/widgets/ActionSetFieldWidget.tid

```diff
@@ -19,7 +19,8 @@ The ''action-setfield'' widget is invisible. Any content within it is ignored.
 |$index |Optional index of a property in a [[data tiddler|DataTiddlers]] to be assigned the $value attribute|
 |$value |The value to be assigned to the field or index identified by the $field or $index attribute. If neither is specified then the value is assigned to the text field. If no value is specified, $field or $index will be deleted.|
 |$timestamp |Specifies whether the timestamp(s) of the target tiddler will be updated (''modified'' and ''modifier'', plus ''created'' and ''creator'' for newly created tiddlers). Can be "yes" (the default) or "no" |
-|//{any attributes not starting with $}// |Each attribute name specifies a field to be modified with the attribute value providing the value to assign to the field  |
+|Attributes of the form TextReference="value" |Each TextReference identifies a field or index to be assigned the corresponding attribute value |
```

Comment:
@matabele, I don't think it can... the html attribute syntax simply doesn't allow for it, unless we invented some markup that looked like `{{reference with blanks!!field}}={{another reference##index}}`.